README.md

```diff
@@ -1 +1,59 @@
-# Mobile-menu
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet) 
+# Mobile-Menu
+
+On this milestone, we have created a menu for our prtfolio for the mobile version. To do this we used Javascript event listeners.
+
+### Built With
+
+![HTML](https://img.shields.io/badge/html5%20-%23E34F26.svg?&style=for-the-badge&logo=html5&logoColor=white)
+![CSS](https://img.shields.io/badge/css3%20-%231572B6.svg?&style=for-the-badge&logo=css3&logoColor=white)
+![JAVASCRIPT](https://img.shields.io/badge/javascript%20-%23323330.svg?&style=for-the-badge&logo=javascript&logoColor=%23F7DF1E")
+
+## Getting Started
+
+## To get local copy up and running follow these simple example steps
+
+```
+git clone https://github.com/AbdumurodovaZulfizar/Mobile-menu.git
+```
+
+```
+cd Mobile-menu
+```
+### Prerequisites
+
+- Any code Editor
+
+### Setup
+
+- If you want to see live version of this project on your local machine install Live Server extension
```

Comment:
You should add more descriptions about how to set up the project in a local environment in order to have a descriptive README file.
**reminder:** It would be nice if any person who doesn't have knowledge in IT can understand it. 😃 

- [ ] Add more instructions to set up the project in a local machine.